aiohttp/client_reqrep.py

```diff
@@ -963,9 +962,7 @@ def links(self) -> "MultiDictProxy[MultiDictProxy[Union[str, URL]]]":
 
             for param in params:
                 match = re.match(r"^\s*(\S*)\s*=\s*(['\"]?)(.*?)(\2)\s*$", param, re.M)
-                if match is None:  # pragma: no cover
-                    # the check exists to suppress mypy error
-                    continue
+                assert match is not None
```

Comment:
This is probably the same.
```suggestion
                if match is None:  # Malformed param
                    continue
```